src/containers/Tabs/Account.tsx

```diff
@@ -120,6 +126,14 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
       };
     }
 
+    let isDuplicateAccount = list.filter((item: Account) => address === item.address).length > 0;
```

Comment:
can use .find !== undefined rather than .filter to avoid always traversing the entire list.